lib/common/session_model.dart

```diff
@@ -73,13 +73,27 @@ class SessionModel extends Model {
   late ValueNotifier<bool?> proxyAvailable;
   late ValueNotifier<String?> country;
 
-  Widget proUser(ValueWidgetBuilder<bool> builder) {
+  Widget proUser(ValueWidgetBuilder<bool> builder, [WebsocketImpl? websocket]) {
     if (isMobile()) {
       return subscribedSingleValueBuilder<bool>('prouser', builder: builder);
     }
     return ffiValueBuilder<bool>(
       'prouser',
       defaultValue: false,
+      onChanges: (setValue) {
+        if (websocket == null) return;
+        /// Listen for all incoming data
+        websocket.messageStream.listen(
+          (json) {
+            if (json["type"] == "pro") {
+              final userStatus = json["message"]["userStatus"];
+              final isProUser = userStatus != null && userStatus.toString() == "active";
```

Comment:
@atavism The problem here is that my userStatus is coming as blank, so my app is not becoming a pro.
And this might happen to another user as well, Do you think we should just change if the type is pro then make it pro.

Reply:
We can check the userLevel as well, but why is your userStatus coming back blank but mine isn't? That's odd. Did you make your user Pro via the panel?

Reply:
Yes, I did make user pro via the pro panel, I asked a query to Relfog, Let's see what we get.

Reply:
Thanks, @jigar-f!